src/components/ManageUsers/manage-users.css

```
@@ -0,0 +1,68 @@
+.user-search__label {
+  margin-left: 4px;
+  font-size: 16px;
+  color: var(--color-primary);
+  font-weight: 400;
+}
+
+.user-search__container {
+  display: flex;
+  flex-direction: column;
+}
+
+.user-search__search {
+  margin: 4px 0;
+  width: 100%;
+  max-width: 600px;
+}
+
+.user-search__input {
+  height: 16px;
+  width: 100%;
+  max-width: 600px;
+  background-color: #242424;
+  color: var(--color-text);
+  border-radius: 8px;
+  padding: 20px;
+  border: none;
+}
+
+.user-search__adduserbutton {
```

here the same rule applies
```suggestion
.user-search__add-user-button {
```